### PR TITLE
ENCM-43 supersedes donor

### PR DESCRIPTION
**Pull request by gabdank**

### CODE REVIEW

File: src/encoded/schemas/fly_donor.json [hunk 1]
```diff
@@ -87,6 +87,18 @@
                 "type":  "string",
                 "pattern": "^(((BDSC|DSSC|VDRC|Harvard|NIG-FLY):.+)|(FlyBase:FBst\\d+)|(GEO:SAMN\\d+))$"
             }
+        },
+        "supersedes": {
+            "title": "Supersedes" ,
+            "description": "The donor(s) that this donor supersedes.",
+            "comment": "The donor(s) in this list should be obsolete and the superseeding donoe object should be used instead.",
```
Comment (ian-whaling) [inline src/encoded/schemas/fly_donor.json]: typo: "superseeding donoe" -> "superseding donor"

**Review by ian-whaling — CHANGES_REQUESTED** [review]
typos: "superseeding donoe" -> "superseding donor" in all comments of all donor schemas

File: src/encoded/schemas/fly_donor.json (hunk 1 above)
Comment (gabdank) [reply src/encoded/schemas/fly_donor.json]: fixed

**Review by ian-whaling — CHANGES_REQUESTED** [review]
"superseding" should only have one e.
https://github.com/ENCODE-DCC/encoded/blob/4c86845b3057d68f4ec8adc4a7f076bffa986674/src/encoded/schemas/analysis.json#L73

**Review by ian-whaling — DISMISSED** [review]

**Pull request merged**